binding.gyp

```diff
@@ -30,23 +30,52 @@
         './include',
         './vendor/agg/include',
         '<(SHARED_INTERMEDIATE_DIR)/',
-        '<!@(pkg-config freetype2 --cflags-only-I | sed s/-I//g)',
-        '<!@(pkg-config protobuf --cflags-only-I | sed s/-I//g)',
+        '<!@(.mason/mason cflags boost 1.58.0 | sed s/-I//g)',
+        '<!@(.mason/mason cflags freetype 2.6 | sed s/-I//g)',
+        '<!@(.mason/mason cflags protobuf 2.6.1 | sed s/-I//g)',
         "<!(node -e \"require('nan')\")"
       ],
       'libraries': [
-        '<!@(pkg-config freetype2 --libs --static)',
-        '<!@(pkg-config protobuf --libs --static)'
+        '<!@(.mason/mason static_libs freetype 2.6)',
+        '<!@(.mason/mason static_libs protobuf 2.6.1)'
+      ],
+      'conditions': [
+        ['OS=="mac"', {
+          'xcode_settings': {
+            'CLANG_CXX_LIBRARY': 'libc++',
+            'GCC_VERSION': 'com.apple.compilers.llvm.clang.1_0',
+            'GCC_ENABLE_CPP_EXCEPTIONS': 'YES',
+            'GCC_ENABLE_CPP_RTTI': 'YES',
+            'OTHER_CPLUSPLUSFLAGS': [
+              '-std=c++1y',
+              '-Wall',
+              '-Wextra',
+              '-Wshadow',
+              '-Wno-variadic-macros',
+              '-Wno-unused-parameter',
+              '-Wno-unused-variable',
```

Comment:
These two are here because of unused variables and parameters in 3rd party libs.